changelogs/fragments/dist_version.yml

```diff
@@ -0,0 +1,2 @@
+bugfixes:
+  - "Various modules and plugins - use vendored version of ``distutils.version`` instead of the deprecated Python standard library ``distutils``."
```

Comment:
Do we really need to embed this? It's pretty easy to compare two Ansible versions with a bit of Python.